jerry-core/ecma/base/ecma-init-finalize.c

```diff
@@ -36,7 +35,6 @@
 void
 ecma_init (void)
 {
-  ecma_lcache_init ();
```

Comment:
Is the cache cleared along the context?

Reply:
Yes, `jerry_init` clears all the context with `memset`. Now that lcache is part of the context, it's initialization is also handled there.